### ISSUE TITLE
Error running Evaluator

### ISSUE THREAD

**Issue by DavidButra** [JdeRobot/DetectionSuite#144]
Hello!

I would like to evaluate some models training with Caffe. Detection Suite starts correctly but when I press over Evaluator the following error shows up:

![Captura de pantalla de 2019-06-18 20-17-58](https://user-images.githubusercontent.com/18295154/59711687-d8bbe980-920b-11e9-81b6-d9b5356e3b35.png)

In the previous image, my configuration file also appears.

Does anyone know what the problem could be?

Thanks!

**Comment by sleep-404** [comment]
Hello @DavidButra ,
That error popped up because the config file might be lacking some required parameter.
Judging from the image , you need to set the  ***inferencesPath*** variable aswell .
Then try running the suite. Hope this helps :)

**Comment by vinay0410** [comment]
Yes @DavidButra , @sleep-404 is right, your yaml config file is incorrect, please post that, so that we can help you correct that.

**Comment by DavidButra** [comment]
Hello @vinay0410, @sleep-404,

Here is my configuration file.

<img width="799" alt="Captura de pantalla 2019-06-19 a las 22 02 30" src="https://user-images.githubusercontent.com/18295154/59796556-125d2500-92de-11e9-91c0-e53bc19597ea.png">

Regards!

**Comment by sleep-404** [comment]
@DavidButra  add 
`inferencesPath: /home/davidbutra/dl-DetectionSuite/DeepLearningSuite/build/DatasetEvaluationApp/datasets`
i.e. similar path as that of ***datasetPath*** to the config file and then run the suite.

**Comment by DavidButra** [comment]
It works!

Thank you @sleep-404, @vinay0410 

Regards!
